_IBC/oraichain-cosmoshub.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "oraichain",
```

Comment:
for all of these, oraichain must be chain_2, since it comes later alphabetically. (same with file names)